src/main/java/nextstep/oauth2/OAuth2AuthenticationFilter.java

```diff
@@ -0,0 +1,122 @@
+package nextstep.oauth2;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.app.domain.Member;
+import nextstep.app.domain.MemberRepository;
+import nextstep.app.infrastructure.InmemoryMemberRepository;
+import nextstep.oauth2.OAuth2ClientProperties.Provider;
+import nextstep.oauth2.OAuth2ClientProperties.Registration;
+import nextstep.security.authentication.UsernamePasswordAuthenticationToken;
+import nextstep.security.context.HttpSessionSecurityContextRepository;
+import nextstep.security.context.SecurityContext;
+import nextstep.security.context.SecurityContextHolder;
+import org.springframework.http.MediaType;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestClient;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+
+public class OAuth2AuthenticationFilter extends OncePerRequestFilter {
+    public static final String OAUTH_BASE_REQUEST_URI = "/login/oauth2/code/";
```

Comment:
```suggestion
    private static final String OAUTH_BASE_REQUEST_URI = "/login/oauth2/code/";
```
상수에 대한 접근제어는 막으면 좋을 것 같아요!

Reply:
반영햇습니다 :)